style.css

```diff
@@ -482,4 +488,476 @@
+  .Mi {
+    font-family: "Poppins", sans-serif;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+    height: 20px;
+    width: 76px;
+    color: #6070ff;
+    list-style-type: none;
+  }
+
+  .burger {
+    display: flex;
+    gap: 100%;
+  }
+
+  .ham {
+    list-style-type: none;
+    display: none;
+  }
+
+  .homesection {
+    background-color: #fff;
+    background-image: url('./img/Header\ bg@2x.png');
+    background-size: 100%;
+    background-repeat: no-repeat;
+    padding-bottom: 1%;
+  }
+
+  .intro {
+    height: 122px;
+    width: 920px;
+    padding-top: 15%;
+    font-family: "Poppins", sans-serif;
+    font-size: 48px;
+    font-weight: 700;
+    line-height: 60px;
+    letter-spacing: 0.3700000047683716px;
+    text-align: left;
+  }
+
+  .intro1 {
+    height: 98px;
+    width: 920px;
+    left: 0;
+    top: 134px;
+    border-radius: nullpx;
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  .social {
+    font-family: "Poppins", sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  .socialicons {
+    display: flex;
+    gap: 2%;
+    margin-left: -30px;
+  }
+
+  .socialicon1 {
+    list-style-type: none;
+  }
+
+  .desktop-nav {
+    display: flex;
+    gap: 20px;
+    justify-content: flex-end;
+    margin-top: -40px;
+  }
+
+  .nav-link {
+    list-style-type: none;
+    font-family: "Poppins", sans-serif;
+    font-size: 15px;
+    font-weight: 500;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+  }
+
+  .desktop-home {
+    margin: 15%;
+    margin-top: -3%;
+  }
+
+  .linkes {
+    text-decoration: none;
+    color: #396df2;
+  }
+
+  .about-head {
+    margin-left: 14px;
+    font-family: "Poppins", sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #172b4d;
+    margin-top: 5px;
+    margin-bottom: -5px;
+  }
+
+  .cano {
+    display: flex;
+    gap: 10%;
+    margin-left: -32px;
+  }
+
+  .tec {
+    font-family: "Poppins", sans-serif;
+    color: #344563;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    height: 24px;
+    width: 75px;
+    left: 0;
+    top: 8px;
+    list-style-type: none;
+  }
+
+  .tech {
+    font-family: "Poppins", sans-serif;
+    color: #6b778c;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    height: 24px;
+    left: 0;
+    top: 8px;
+    border-radius: nullpx;
+  }
+
+  .about-section {
+    display: flex;
+    gap: 75px;
+    border: 1px solid #cfd8dc;
+    padding: 20px;
+    border-radius: 16px;
+    width: 85%;
+    margin: 6%;
+  }
```

Comment:
As earlier mentioned, please stick as closely as possible to the Figma design as possible. 🎯 

- [ ] I suggest you chack your padding and margin units and use responsive units like `%`, `em`, `rem` etc.

Kindly compare the images below and center your works as seen in the design template ♻️ 

### Your design ⬇️ 
![image](https://github.com/Danny-imo/my-project/assets/117144028/25ae79b4-1517-4ca4-9958-0ff33867bee3)

### Figma design ⬇️ 
![image](https://github.com/Danny-imo/my-project/assets/117144028/268d3335-689f-4297-9ce0-9d2c93bae329)